src/main/kotlin/io/github/yuutoproject/yuutobot/commands/minigame/MinigameInstance.kt

```diff
@@ -0,0 +1,185 @@
+/*
+ * Open source bot built by and for the Camp Buddy Discord Fan Server.
+ *     Copyright (C) 2020  Kyuuto-devs
+ *
+ *     This program is free software: you can redistribute it and/or modify
+ *     it under the terms of the GNU General Public License as published by
+ *     the Free Software Foundation, either version 3 of the License, or
+ *     (at your option) any later version.
+ *
+ *     This program is distributed in the hope that it will be useful,
+ *     but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *     GNU General Public License for more details.
+ *
+ *     You should have received a copy of the GNU General Public License
+ *     along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package io.github.yuutoproject.yuutobot.commands.minigame
+
+import io.github.yuutoproject.yuutobot.commands.Minigame
+import io.github.yuutoproject.yuutobot.objects.Question
+import net.dv8tion.jda.api.EmbedBuilder
+import net.dv8tion.jda.api.entities.User
+import net.dv8tion.jda.api.events.message.guild.GuildMessageReceivedEvent
+import net.dv8tion.jda.api.events.message.guild.react.GuildMessageReactionAddEvent
+import net.dv8tion.jda.api.events.message.guild.react.GuildMessageReactionRemoveEvent
+import java.awt.Color
+
+enum class State {
```

Comment:
This will be put in a separate file when compiled btw

Reply:
I was considering removing it anyway. Right now there's only two states, so I can just replace it with a `gameStarted: bool` variable.